### PR TITLE
howework on Streams added

### PR DESCRIPTION
**Pull request by turandot42**
Streams hw added

### CODE REVIEW

File: 02-GenericsAndLambda/src/main/java/LambdaJava1.java [hunk 1]
```diff
@@ -17,10 +21,23 @@ public static String getSlogan(DummyInterface dummyInterface) {
...
     }
 }
+
+class DevX implements  DummyInterface{
+    @Override
+    public String applySlogan(String slogan) {
+        return slogan+ "Developing experience";
+    }
+}
+
```
Comment (gantu) [inline 02-GenericsAndLambda/src/main/java/LambdaJava1.java:40]: Try to not leave unnecessary  empty lines.

File: 02-GenericsAndLambda/src/main/java/homework/adina/StreamTasks.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+        //task2 print
+        Stream<String> info= Stream.of("2","3","Jack");
+        Optional<String> name=info.filter(a->a.equals("2")).findFirst();
+        System.out.println(name);
+
+//task3
+        String s= "The quick brown fox and the quick blue hare";
+//        String st=s.substring(0,s.indexOf("the"));
```
Comment (gantu) [inline 02-GenericsAndLambda/src/main/java/homework/adina/StreamTasks.java:38]: confusing comments should be removed

File: 02-GenericsAndLambda/src/main/java/LambdaJava1.java (hunk 1 above)
Comment (turandot42) [reply 02-GenericsAndLambda/src/main/java/LambdaJava1.java:39]: Done @gantu

File: 02-GenericsAndLambda/src/main/java/homework/adina/StreamTasks.java (hunk 2 above)
Comment (turandot42) [reply 02-GenericsAndLambda/src/main/java/homework/adina/StreamTasks.java]: I was playing around with String, good catch, removed it @gantu

**Pull request merged**